pkg/gui/custom_commands.go

```diff
@@ -52,8 +54,11 @@ func (gui *Gui) createCommandMenu(customCommands []config.CustomCommand, command
 
 		// if we have a command for attaching, we attach and return the subprocess error
 		if option.customCommand.Attach {
-			cmd := gui.OSCommand.ExecutableFromString(option.command)
-			return gui.runSubprocess(cmd)
+			if option.invokeShell {
+				return gui.runSubprocess(gui.OSCommand.ExecutableFromStringWithShell(option.command))
+			}
+
+			return gui.runSubprocess(gui.OSCommand.ExecutableFromString(option.command))
```

Comment:
Also renamed the func to `ExecutableFromStringWithShell` to follow the pattern of `Executable` funcs on `pkg/commands/os.go`